src/main/java/com/google/devtools/build/lib/authandtls/staticcredentials/StaticCredentials.java

```diff
@@ -0,0 +1,51 @@
+package com.google.devtools.build.lib.authandtls.staticcredentials;
+
+import com.google.auth.Credentials;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+/** Implementation of {@link Credentials} which provides a static set of credentials. */
+public final class StaticCredentials extends Credentials {
+  private final ImmutableMap<URI, Map<String, List<String>>> credentials;
+
+  public StaticCredentials(Map<URI, Map<String, List<String>>> credentials) {
+    Preconditions.checkNotNull(credentials);
+
+    this.credentials = ImmutableMap.copyOf(credentials);
+  }
+
+  public Map<URI, Map<String, List<String>>> getMapForMigration() {
+    return credentials;
+  }
```

Comment:
I couldn't find any callers for this method in this PR. Will it be used in a followup? If so, consider adding it in the same PR where it gets used, so we can later trace back why it was added.

Reply:
Eh, yeah. I used this when creating this PR. Removing